extensions/walletProviders/kadena/src/extension.ts

```diff
@@ -0,0 +1,79 @@
+import { ITatumSdkContainer, Network, TatumConfig, TatumSdkWalletProvider } from '@tatumio/tatum'
+import { KadenaNetworkId, KadenaTxPayload, KadenaWallet } from './types'
+import { genKeyPair } from "@kadena/cryptography-utils/lib/genKeyPair";
+import { restoreKeyPairFromSecretKey } from "@kadena/cryptography-utils";
+import { ChainId, createClient, createSignWithKeypair, ICommand } from "@kadena/client";
+import { ICommandPayload } from "@kadena/types";
+import { KadenaLoadBalancerRpc } from "@tatumio/tatum/dist/src/service/rpc/other/KadenaLoadBalancerRpc";
```

Comment:
GLOBAL: Again, this is a nitpick, but it would be nice to have imports form external libraries at the top.

Reply:
Fixed